app/config/errors.php

```diff
@@ -474,6 +474,11 @@
         'description' => 'Project with the requested ID could not be found. Please check the value of the X-Appwrite-Project header to ensure the correct project ID is being used.',
         'code' => 404,
     ],
+    Exception::ENV_VARIABLE_INCORRECT_SETUP => [
+        'name' => Exception::ENV_VARIABLE_INCORRECT_SETUP,
+        'description' => 'For Custom Domain ENV Variable is not setup correctly',
```

Comment:
Let's replace this with:

```suggestion
        'description' => 'Your Appwrite instance is not publicly accessible. Please check the _APP_DOMAIN_TARGET environment variable of your Appwrite server.',
```

Please also make sure to put this error under the `DOMAIN_VERIFICATION_FAILED ` error.